spinn_front_end_common/utility_models/reverse_ip_tag_multicast_source_machine_vertex.py

```diff
@@ -534,6 +536,11 @@ def _write_configuration(self, spec, n_machine_time_steps):
         # write SDP port to which SDP packets will be received
         spec.write_value(data=self._receive_sdp_port)
 
+        # write random tx offset in microseconds
+        import numpy as np
+        tx_offset = 0.5 * np.random.rand()
```

Comment:
For reproducibility, the random source needs to be seedable and independent of other uses of random numbers.